src/OrbitGl/App.h

```diff
@@ -511,6 +511,17 @@ class OrbitApp final : public DataViewFactory,
   [[nodiscard]] const orbit_statistics::BinomialConfidenceIntervalEstimator&
   GetConfidenceIntervalEstimator() const override;
 
+  void SetHistogramSelectionRangeChange(
```

Comment:
Nit: What about calling it `SetHistogramSelectionRange`. The `Change` does not really makes sense in my opinion when there is a `Set` at the front.